src/gsi/get_gefs_for_regional.f90

```diff
@@ -540,9 +708,29 @@ subroutine get_gefs_for_regional
               work_sub(1,i,j,kq)=q(i,j,k)
               work_sub(1,i,j,koz)=oz(i,j,k)
               work_sub(1,i,j,kcw)=cwmr(i,j,k)
+!              if ( tv(i,j,k) .lt. 0. ) then
```

Comment:
Remove these lines if they are not needed.

Reply:
Removed.